src/devices/machine/laserdsc.h

```diff
@@ -335,6 +335,25 @@ typedef device_interface_enumerator<laserdisc_device> laserdisc_device_enumerato
 //  INLINE FUNCTIONS
 //**************************************************************************
 
+// ======================> parallel_laserdisc_device
+
+class parallel_laserdisc_device : public laserdisc_device
+{
+public:
+
+	virtual void data_w(u8 data) = 0;
+	virtual u8 data_r() = 0;
+	virtual void enter_w(int state) {}
+	virtual int data_available_r() { return CLEAR_LINE; }
+	virtual int status_strobe_r() { return CLEAR_LINE; }
+	virtual int ready_r() { return ASSERT_LINE; }
+
+protected:
+	parallel_laserdisc_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, u32 clock = 0);
+
+	virtual void player_overlay(bitmap_yuy16 &bitmap) override { }
+};
```

Comment:
I get that this is to simplify supporting multiple player types in a single driver.  But how artificial is it to make the players with parallel interfaces conform to a single interface?  (Honest question, I haven’t really looked at them.  I’m more familiar with Pioneer’s later “System Remote” interfaces for controlling players.)

The other alternative to avoid piles of `if (m_optional_player_device) ...` would be to have delegates or devcbs in the driver and hook them up in the machine configuration function when the player is added.

Reply:
A bit artificial, but not completely. Two of the lines are mutually exclusive between the LD-V1000 and LD-PR7820. I don't feel strongly one way or another about it, but I'm leaning towards letting it ride until we can either change the player type based on machine-configuration switches, and there's working player LLE (or working LD-PR7280 HLE).